package.json

```diff
@@ -22,6 +22,12 @@
       "import": "./esm/vanilla.mjs",
       "default": "./vanilla.js"
     },
+    "./setup-tests": {
```

Comment:
let me review this and `scripts` later.

Reply:
I think this entry point is jest specific. So, can it be like `zustand/setup-jest`?
Do you have any prior art like this pattern? I'd like to know how common it is.

Just wondering if people would like something more explicit?
```js
import { mockStore } from 'zustand/setup-jest';
mockStore();
```
This is not a suggestion by any means.

Reply:
im testing out if this works atm, will post PR with all changes in couple of hours probably

but most of the patterns for jest are `import 'jest-dom/setup';`

Reply:
reference is here: https://testing-library.com/docs/dom-testing-library/example-intro

`import '@testing-library/jest-dom'`